<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FUP Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_serverIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upload IP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIP">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ip">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8080</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_serverPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPort">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFile">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textLog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonUpload</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonIP</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setIP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPort</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setPort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>upload()</slot>
  <slot>setIP()</slot>
  <slot>setPort()</slot>
  <slot>openFile()</slot>
 </slots>
</ui>
